src/pages/ExamsPage/components/ExternalReviewDashboard.jsx

```diff
@@ -16,12 +16,16 @@ const ExternalReviewDashboard = ({ exam }) => {
   return (
     <div data-testid="review_dash">
       <div style={{ padding: 10 }}>
-        {!ltiToolEmbed && exam && (
-          <Button as="a" title="lti_link" target="_blank" href={getLaunchUrlByExamId(exam.id)}>
-            {formatMessage(messages.ReviewDashboardOpenLTITool)}
-            <Launch />
-          </Button>
-        )}
+        {
+          // If an exam is selected, show the button to open the external review dashboard,
+          // otherwise prompt the user to select an exam.
+          (!ltiToolEmbed && exam) ? (
+            <Button as="a" title="lti_link" target="_blank" href={getLaunchUrlByExamId(exam.id)}>
+              {formatMessage(messages.ReviewDashboardOpenLTITool) + exam.name}
```

Comment:
Instead of string concatenation, I think you should do something similar to https://github.com/openedx/frontend-app-learning/blob/d065c23e32763a5dca0eb32c5cc7723769c9d72b/src/alerts/sequence-alerts/hooks.js#L42-L45 and https://github.com/openedx/frontend-app-learning/blob/d065c23e32763a5dca0eb32c5cc7723769c9d72b/src/alerts/sequence-alerts/messages.js#L4. 

This is important because not all languages use the same word order/syntax structure as english, so using the approach above allows for a correct translation :)